site/index.md

```diff
@@ -63,15 +63,15 @@ getInvolvedSteps:
 
 ## About StarlingX
 
-StarlingX is a complete cloud infrastructure software stack for the edge used by the most demanding applications in industrial IOT, telecom, video delivery and other ultra-low latency use cases. Based on mature software deployed for mission critical applications, newly open sourced StarlingX code is the base for edge implementations in scalable solutions that is ready for production now.
+StarlingX is a complete cloud infrastructure software stack for the edge used by the most demanding applications in industrial IOT, telecom, video delivery and other ultra-low latency use cases. With deterministic low latency required by edge applications, and tools that make distributed edge manageable, StarlingX provides a container-based infrastructure for edge implementations in scalable solutions that is ready for production now.
 
 <a href="/learn/" class="link is-primary">Learn More ></a>
 
 </template>
 
 <home-announcement slot="announcement" button-name="Read the blog post" link="/blog/starlingx-initial-release.html">
 
-StarlingX—the open source edge computing and IoT cloud platform optimized for low latency and high performance applications—is now available in its first release. The project was established in May as a pilot project supported by the OpenStack Foundation (OSF) and builds on code contributed by Wind River and Intel.
+StarlingX Release 2.0 is now available. The open source edge computing and IoT cloud platform optimized for low latency and high performance applications is now a container based platfom for hosting OpenStack and other cloud native applications.
 
```

Comment:
s/platfom/platform/

I wonder if we could rephrase this "a container based platform for hosting OpenStack and other cloud native applications" to show integration between OpenStack and the container platform?

Reply:
Done.

Reply:
Came up with an alternative to the second sentence:

The open source edge computing and IoT cloud platform optimized for low latency and high performance applications now integrates OpenStack with container technologies to provide flexibility, robustness and support for mixed workloads.